idd/Energy+.idd.in

```diff
@@ -1779,26 +1898,15 @@ Site:GroundDomain:Slab,
         \minimum 0
         \maximum 100
         \default 50
-   N9,  \field Kusuda-Achenbach Average Ground Surface Temperature
-        \type real
-        \units C
-        \note Annual average ground surface temperature.
-        \note If left blank, Site:GroundTemperature:Shallow must be used to provied
-        \note monthly average ground surface temperatures.
-   N10, \field Kusuda-Achenbach Average Amplitude of Ground Surface Temperature
-        \type real
-        \units deltaC
-        \note Annual average ground surface temperature variation from average.
-        \note If left blank, Site:GroundTemperature:Shallow must be used to provied
-        \note monthly average ground surface temperatures.
-   N11, \field Kusuda-Achenbach Phase Shift of Minimum Ground Surface Temperature
-        \type real
-        \units days
-        \note The phase shift of minimum ground surface temperature, or the day
-        \note of the year when the minimum ground surface temperature occurs.
-        \note If left blank, Site:GroundTemperature:Shallow must be used to provied
-        \note monthly average ground surface temperatures.
-   N12, \field Evapotranspiration Ground Cover Parameter
+   A2,  \field Type of Undisturbed Ground Temperature Model
```

Comment:
Common pattern it to put "Type" and "Name" and the end of the field name.  e.g. Undisturbed Ground Temperature Model Type and Undisturbed Ground Temperature Model Name (symmetrical fields and no formal type names in the field name).